circle.yml

```diff
@@ -4,37 +4,32 @@ machine:
 
 dependencies:
   override:
-    - mvn -U dependency:resolve dependency:resolve-plugins
```

Comment:
Please check the [CircleCI docs](https://circleci.com/docs/language-java/#dependencies--tests): It is `./gradlew dependencies`.